CMakeLists.txt

```diff
@@ -355,6 +356,7 @@ add_executable(pvtx-static
 	${PVTX_COMMON_SOURCES}
 )
 target_compile_definitions(pvtx-static PUBLIC -DDISABLE_LOGSERVER)
+target_compile_definitions(pvtx PUBLIC -DPV_FS_NO_SYNC)
```

Comment:
why do we need this here? is that something we turn on for pvtx build, but not on for pantavisor itself? whats the rational to not have two functions? ... or a wrapper function that does the sync for the sync variant...